tests/iterator/tree.c

```diff
@@ -1008,8 +1013,11 @@ void test_iterator_tree__pathlist_with_directory(void)
 	i_opts.pathlist.strings = (char **)filelist.contents;
 	i_opts.pathlist.count = filelist.length;
 
+	const char *expected2[] = { "subdir/subdir2/README", "subdir/subdir2/new.txt" };
+	size_t expected_len2 = 2;
```

Comment:
Please move declarations up to the top of the block so that old MSVC doesn't choke on it.  :(